X/XicTools/build_tarballs.jl

```diff
@@ -0,0 +1,106 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "XicTools"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/wrcad/xictools", "c7a50a5fcd71966730e45a5358b8507227ae098c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/xictools/
+cp Makefile.sample Makefile
+sed -i "s|enable-itopok=no|enable-itopok=yes|g" Makefile
+sed -i "s|GFXLOC = --enable-gtk2=yes|#GFXLOC = --enable-gtk2=yes|g" Makefile
+#sed -i "s|^SUBDIRS =.*|SUBDIRS = xt_base adms KLU vl wrspice|g" Makefile
+sed -i "766c\#include <malloc.h>" xt_base/malloc/malloc-2.8.6.c
+update_configure_scripts
+make config
+make all -j$(nproc)
+make prefix="${prefix}/usr" install
+install_license ${WORKSPACE}/srcdir/xictools/license/LICENSE-2.0.txt
+cd ${prefix}/usr/xictools
```

Comment:
Should we delete the files that you're copying from here below? Otherwise we have duplicate files in the tarball, no?

Reply:
Good point, thank you, yes, I do see those duplicated in the tarballs. I'll give deleting them a shot and test if the program works as expected (or try to get BinaryBuilder to recognize those executables without copying).

Reply:
I made these changes, tested locally and it looks ok, and it appears to successfully build on buildkite and pass all the checks on github. If there is anything else to do on my end before merging, please let me know. Thanks!